jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -70,9 +70,9 @@ private String getFavoriteEditor() {
     }
     
     // A diagnostic will appear as @JsonbTransient is not mutually exclusive on this accessor
-    @JsonbNillable
+    @JsonbAnnotation
```

Comment:
Was `@JsonbNillable` incorrectly used here?

Reply:
Yes, it's not a method level annotation